apiclient/container_inspect.go

```diff
@@ -0,0 +1,748 @@
+	Env []string `json:"Env"`
+	// Container command
+	Cmd []string `json:"Cmd"`
+	// Container image
+	Image string `json:"Image"`
+	// Unused, at present. I've never seen this field populated.
+	Volumes map[string]struct{} `json:"Volumes"`
+	// Container working directory
+	WorkingDir string `json:"WorkingDir"`
+	// Container entrypoint
+	Entrypoint string `json:"Entrypoint"`
+	// On-build arguments - presently unused. More of Buildah's domain.
+	OnBuild *string `json:"OnBuild"`
+	// Container labels
+	Labels map[string]string `json:"Labels"`
+	// Container annotations
+	Annotations map[string]string `json:"Annotations"`
+	// Container stop signal
+	StopSignal uint `json:"StopSignal"`
+	// Configured healthcheck for the container
+	//Healthcheck *manifest.Schema2HealthConfig `json:"Healthcheck,omitempty"`
+	// CreateCommand is the full command plus arguments of the process the
+	// container has been created with.
+	CreateCommand []string `json:"CreateCommand,omitempty"`
+	// Timezone is the timezone inside the container.
+	// Local means it has the same timezone as the host machine
+	Timezone string `json:"Timezone,omitempty"`
+	// Umask is the umask inside the container.
+	Umask string `json:"Umask,omitempty"`
+	// SystemdMode is whether the container is running in systemd mode. In
+	// systemd mode, the container configuration is customized to optimize
+	// running systemd in the container.
+	SystemdMode bool `json:"SystemdMode,omitempty"`
+	// Unused, at present
+	AttachStdin bool `json:"AttachStdin"`
+	// Unused, at present
+	AttachStdout bool `json:"AttachStdout"`
+	// Unused, at present
+	AttachStderr bool `json:"AttachStderr"`
+	// Whether the container creates a TTY
+	Tty bool `json:"Tty"`
+	// Whether the container leaves STDIN open
+	OpenStdin bool `json:"OpenStdin"`
+	// Whether STDIN is only left open once.
+	// Presently not supported by Podman, unused.
+	StdinOnce bool `json:"StdinOnce"`
+}
+
+// InspectRestartPolicy holds information about the container's restart policy.
+type InspectRestartPolicy struct {
+	// Name contains the container's restart policy.
+	// Allowable values are "no" or "" (take no action),
+	// "on-failure" (restart on non-zero exit code, with an optional max
+	// retry count), and "always" (always restart on container stop, unless
+	// explicitly requested by API).
+	// Note that this is NOT actually a name of any sort - the poor naming
+	// is for Docker compatibility.
+	Name string `json:"Name"`
+	// MaximumRetryCount is the maximum number of retries allowed if the
+	// "on-failure" restart policy is in use. Not used if "on-failure" is
+	// not set.
+	MaximumRetryCount uint `json:"MaximumRetryCount"`
+}
+
+// InspectLogConfig holds information about a container's configured log driver
+// and is presently unused. It is retained for Docker compatibility.
+type InspectLogConfig struct {
+	Type   string            `json:"Type"`
+	Config map[string]string `json:"Config"` //idk type, TODO
```

Comment:
:sweat_smile: took me a moment to realize this //idk type, TODO is from upstream